rest/changes_api_test.go

```diff
@@ -556,6 +546,158 @@ func postChangesAdminChannelGrant(t *testing.T, rt *RestTester) {
 
 }
 
+func TestPostChangesAdminChannelGrantRemoval(t *testing.T) {
+
+	if testing.Short() {
```

Comment:
This test seems quick enough (particularly with no sleep) - should be ok to remove this?

Reply:
Sure Adam, let's remove this.